mllib/src/main/scala/org/apache/spark/ml/feature/StringIndexer.scala

```diff
@@ -131,6 +163,12 @@ object StringIndexer extends DefaultParamsReadable[StringIndexer] {
   private[feature] val KEEP_INVALID: String = "keep"
   private[feature] val supportedHandleInvalids: Array[String] =
     Array(SKIP_INVALID, ERROR_INVALID, KEEP_INVALID)
+  private[feature] val FREQ_DESC: String = "freq_desc"
```

Comment:
@felixcheung I did not find any prior standard, and am open to suggestion for better names. 
Maybe better use `frequency_desc` or `count_desc`?